### ISSUE TITLE
atom_diag python example doc

### ISSUE THREAD

**Issue by mzingl** [TRIQS/triqs#702]
Dear all,
the python example in the atom_diag doc:
https://triqs.github.io/triqs/2.1.x/reference/atom_diag/contents.html
does not work for the calculation of Green functions.
I suggest to change the example below the line '# Atomic Matsubara Green's function'
to
```
# Atomic Green's functions
gf_struct = [['dn',orb_names],['up',orb_names]]
G_w = atomic_g_w(ad, beta, gf_struct, (-2, 2), 400, 0.01)
G_tau = atomic_g_tau(ad_r, beta, gf_struct, 400)
G_iw = atomic_g_iw(ad_r, beta, gf_struct, 100)
G_l = atomic_g_l(ad_r, beta, gf_struct, 20)

# Finally, we save our AtomDiag object for later use
with HDFArchive('atom_diag_example.h5') as ar:
    ar['ad'] = ad
```
Best,
Manuel

**Comment by Wentzell** [comment]
Hey @mzingl 
Thank you for noticing this broken example!
In your correction, can you add the definition of `ad_r` ?

**Comment by mzingl** [comment]
Hi @Wentzell 
I'm sorry, ad_r should also be ad:
```
# Atomic Green's functions
gf_struct = [['dn',orb_names],['up',orb_names]]
G_w = atomic_g_w(ad, beta, gf_struct, (-2, 2), 400, 0.01)
G_tau = atomic_g_tau(ad, beta, gf_struct, 400)
G_iw = atomic_g_iw(ad, beta, gf_struct, 100)
G_l = atomic_g_l(ad, beta, gf_struct, 20)

# Finally, we save our AtomDiag object for later use
with HDFArchive('atom_diag_example.h5') as ar:
    ar['ad'] = ad
```

**Comment by Wentzell** [comment]
Fixed in 71db2f1c
